security/tls-authorization/README.adoc

```diff
@@ -0,0 +1,226 @@
+= Remote caches with TLS (SSL) encryption and security authorization
+
+**Authors:** Wolf Fink +
+**Technologies:** Infinispan, Hot Rod, Java, SSL/TLS, RBAC +
+**Summary:** Use SSL/TLS certificates to verify Hot Rod client identities and encrypt network traffic to the server. This tutorial also demonstrates how to perform client certificate authentication and use security authorization with client certificates.
+
+== Building instructions and preparation
+
+. `$ mvn clean package`
+. Build the example certificates and keystores with the following scripts or directly with the Java keytool commands:
+* `create_unsigned_certificates.sh`
+* `create_signed_certificates.sh`
++
+These scripts create keystores and trust stores in the project directory.
```

Comment:
@wfink What about a script to clean the filesystem and delete all the .cer and .pfx artifacts?

Reply:
"git clean" should do the job

Reply:
yeah, I was also thinking about the server installation. but maybe we should just leave that to users. maybe we should suggest creating a server instance that the user can simply remove after running the tutorial?